updater/internal/service/service_linux.go

```diff
@@ -59,39 +78,45 @@ func NewService(logger *zap.Logger, installDir string, opts ...Option) Service {
 	return linuxSvc
 }
 
-type linuxService struct {
+type linuxService interface {
+	Service
+	setNewSvcFile(string)
+}
+
+type linuxSystemdService struct {
 	// newServiceFilePath is the file path to the new unit file
 	newServiceFilePath string
 	// serviceName is the name of the service
-	serviceName string
+	serviceName    string
+	serviceCmdName string
```

Comment:
I think we can remove this variable now that we have 2 separate structs (service command name should always be "systemctl" for systemd, "service" for sysv)